adk/server/server.go

```diff
@@ -19,6 +19,13 @@ import (
 	zap "go.uber.org/zap"
 )
 
+// Build-time metadata variables set via LD flags
+var (
+	BuildAgentName        = "helloworld-agent"
+	BuildAgentDescription = "A simple greeting agent that provides personalized greetings using the A2A protocol"
+	BuildAgentVersion     = "1.0.0"
+)
+
```

Comment:
Can we use a separate file for this metadata ? maybe call it metadata.go for clarity ?